lib/jobs/install-os.js

```diff
@@ -97,13 +97,20 @@ function installOsJobFactory(
                         }
                         /* jshint ignore:end */
                     });
-                    if(dev.ipv4.vlanId) {
-                        vlanIds = dev.ipv4.vlanId;
+
+                    //support vlanId with 'deprecated' message
+                    if(dev.ipv4.vlanId){
+                        dev.ipv4.vlanIds = dev.ipv4.vlanId;
```

Comment:
Comparing `vlanIds` and 'vlanId`, The `vlanIds` should take precedent, so `vlanId` is used only if `vlanIds` is not found.